crates/sncast/tests/e2e/invoke.rs

```diff
@@ -267,78 +275,45 @@ fn test_too_low_gas() {
         output,
         indoc! {r"
         command: invoke
-        error: Max fee is smaller than the minimal transaction cost
+        error: The transaction's resources don't cover validation or the minimal transaction fee
         "},
     );
 }
 
-#[test]
-fn test_max_gas_equal_to_zero() {
-    let args = vec![
-        "--accounts-file",
-        ACCOUNT_FILE_PATH,
-        "--account",
-        "user11",
-        "--wait",
-        "invoke",
-        "--url",
-        URL,
-        "--contract-address",
-        MAP_CONTRACT_ADDRESS_SEPOLIA,
-        "--function",
-        "put",
-        "--calldata",
-        "0x1",
-        "0x2",
-        "--max-gas",
-        "0",
-    ];
-
-    let snapbox = runner(&args);
-    let output = snapbox.assert().code(2);
-
-    assert_stderr_contains(
-        output,
-        indoc! {r"
-        error: invalid value '0' for '--max-gas <MAX_GAS>': Value should be greater than 0
-        "},
-    );
-}
-
-#[test]
-fn test_calculated_max_gas_equal_to_zero_when_max_fee_passed() {
-    let args = vec![
-        "--accounts-file",
-        ACCOUNT_FILE_PATH,
-        "--account",
-        "user11",
-        "--wait",
-        "invoke",
-        "--url",
-        URL,
-        "--contract-address",
-        MAP_CONTRACT_ADDRESS_SEPOLIA,
-        "--function",
-        "put",
-        "--calldata",
-        "0x1",
-        "0x2",
-        "--max-fee",
-        "999999",
-    ];
-
-    let snapbox = runner(&args);
-    let output = snapbox.assert().success();
-
-    // TODO(#2852)
-    assert_stderr_contains(
-        output,
-        indoc! {r"
-        command: invoke
-        error: Calculated max-gas from provided --max-fee and the current network gas price is 0. Please increase --max-fee to obtain a positive gas amount: Tried to create NonZeroFelt from 0
-        "},
-    );
-}
+// #[test]
+// fn test_calculated_max_gas_equal_to_zero_when_max_fee_passed() {
+//     let args = vec![
+//         "--accounts-file",
+//         ACCOUNT_FILE_PATH,
+//         "--account",
+//         "user11",
+//         "--wait",
+//         "invoke",
+//         "--url",
+//         URL,
+//         "--contract-address",
+//         MAP_CONTRACT_ADDRESS_SEPOLIA,
+//         "--function",
+//         "put",
+//         "--calldata",
+//         "0x1",
+//         "0x2",
+//         "--max-fee",
+//         "999999",
+//     ];
+
+//     let snapbox = runner(&args);
+//     let output = snapbox.assert().success();
+
+//     // TODO(#2852)
+//     assert_stderr_contains(
+//         output,
+//         indoc! {r"
+//         command: invoke
+//         error: Calculated max-gas from provided --max-fee and the current network gas price is 0. Please increase --max-fee to obtain a positive gas amount: Tried to create NonZeroFelt from 0
+//         "},
+//     );
+// }
```

Comment:
Uncomment (and ignore with a linked issue if needed). Or remove if it is no longer relevant (kind of seems like it).